bookwyrm/management/commands/remove_remote_user_preview_images.py

```diff
@@ -0,0 +1,40 @@
+""" Remove preview images for remote users """
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from bookwyrm import models, preview_images
+
+
+# pylint: disable=line-too-long
+class Command(BaseCommand):
+    """Remove preview images for remote users"""
+
+    help = "Remove preview images for remote users"
+
+    # pylint: disable=no-self-use,unused-argument
+    def handle(self, *args, **options):
+        """generate preview images"""
+        self.stdout.write(
+            "   | Hello! I will be removing preview images from remote users."
+        )
+        self.stdout.write(
+            "🧑‍🚒 ⎨ This might take quite long if your instance has a lot of remote users."
+        )
+        self.stdout.write("   | ✧ Thank you for your patience ✧")
+
+        users = models.User.objects.filter(local=False).exclude(
+            Q(preview_image="") | Q(preview_image=None)
+        )
```

Comment:
This query is quite power-intensive. As this command is destined to be ran a couple of times at most in the life of the instance, I don't know if it's worth being optimized

Reply:
I think it's fine as is, given, like you said, that it's not meant ot be in regular use